snuba/api.py

```diff
@@ -344,55 +348,45 @@ def sdk_distribution(validated_body, timer):
 if application.debug or application.testing:
     # These should only be used for testing/debugging. Note that the database name
     # is checked to avoid scary production mishaps.
-    assert settings.CLICKHOUSE_TABLE in ('dev', 'test')
 
-    _ensured = False
+    _ensured = {}
 
-    def ensure_table_exists(force=False):
+    def ensure_table_exists(dataset, force=False, drop=False):
```

Comment:
nit: what about forcing the caller to pass force and drop as named arguments (if they pass them). I think it is something like

> def ensure_table_exists(dataset, *, force=False, drop=False):

so you won't see around the code things like 

> ensure_table_exists("something", True, False)
> ensure_table_exists("something", False, False)

fewer chances of mixing up the two booleans